radio/src/telemetry/flysky_nv14.cpp

```diff
@@ -125,30 +125,41 @@ int32_t GetSensorValueFlySkyNv14(const FlyskyNv14Sensor* sensor,
   int32_t value = 0;
   const nv14SensorData* sensorData =
       reinterpret_cast<const nv14SensorData*>(data + sensor->offset);
+  static bool oldRmFm = false;
```

Comment:
I would prefer if this was less vague... older than what version? ;) `fwPreVXYZ` or something along  those lines maybe?

Reply:
The oldest ever available. :-)
I will change it.